align/pnr/manipulate_hierarchy.py

```diff
@@ -83,7 +83,7 @@ def modify_pg_conn_subckt(verilog_d, subckt, pp):
     """
     # TODO: remove redundant const
 
-    nm = copy.copy([module for module in verilog_d['modules'] if module['name'] == subckt][0])
+    nm = copy.deepcopy([module for module in verilog_d['modules'] if module['name'] == subckt][0])
```

Comment:
Is deepcopy really needed here? deepcopy on `VerilojJson` objects is expensive.

Reply:
@soneryaldiz Can you please check if tests/pdk/finfet_pdk/test_placement_grid.py::test_bias_generator passes on your personal system? It passes on mine.

Reply:
@kkunal1408 , it passes locally for me as well but it is taking a very long time. Let me revise the test.

Reply:
deepcopy is blowing up the memory consumption (up to 15GB). We need another way to manipulate beyond deepcopy.